### ISSUE TITLE
Use SWGOH.GG for recruit thread data?

### ISSUE THREAD

**Issue by veritablequandary** [veritablequandary/senatebot#16]
**Description**
Currently using SWGOH.help real-time client for pulling game data. This means whatever data I display is up to date within a couple of seconds.

Have debated using the new SWGOH.gg public API for a few reasons. Mainly, fleet arena rank shown is from last payout, which is more useful info than current rank. SWGOH.GG shows GL ultimates and Omicrons.

Downside of using SWGOH.gg is a) information isn't quite as current and b) if the user doesn't have a GG profile almost no information would be displayed. Need to decide if those downsides are offset by the potential benefits.

**Comment by Terebi42** [comment]
re the last issue, pull .gg and if it fails fall back to .help?

**Comment by veritablequandary** [comment]
> re the last issue, pull .gg and if it fails fall back to .help?

Yes! I think that's what I'm going with. Will probably use the embed footer to show which data source was used (and when it was last synced, in the case of .gg)
